tests/ff_module-task-meta-actions.test.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+var React = require('react');
+
+var TestUtils = require('react-addons-test-utils'),
+    expect = require('chai').expect;
+
+var TaskMetaActions = require('../blocks/core/ff_module/ff_module-task-meta-actions/ff_module-task-meta-actions'),
+    Button = require('../blocks/core/ff_module/ff_module-button/ff_module-button'),
+    DropdownButton = require('../blocks/core/ff_module/ff_module-dropdown-button/ff_module-dropdown-button-component/ff_module-dropdown-button-component'),
+    DropdownFilters = require('../blocks/core/ff_module/ff_module-dropdown-filters/ff_module-dropdown-filters'),
+
+    getElementByClass = require('./lib/framework').setupGetElementByClass(React, TestUtils, TaskMetaActions);
+
+var props = [{
+    state: {
+        // archived: true
+    },
+    description: {
+        numRecipientsAffected: 43,
+        author: { name: 'Terry Teacher' }
+    },
+    filters: {
+        text: '[Filter by Status]',
+        onAddFilter: function(id, event) { console.log('Adding ' + id); },
+        onRemoveFilter: function(id, event) { console.log('Removing ' + id); },
+        filters: [{
+            name: '[Awaiting Response]',
+            id: 'filter-1'
+        }, {
+            isActive: true,
+            name: '[Approved]',
+            id: 'filter-2'
+        }, {
+            name: '[Response Received]',
+            id: 'filter-3'
+        }]
+    },
+    singleActions: [{ text: '[Send All Now]', onClick: function() {} }],
+    groupedActions: {
+        text: '[More Actions]',
+        list: [{ text: '[Edit]', onClick: function(event) { console.log(event); } }]
+    }
+}, {
+    state: {
```

Comment:
There's another state object here https://github.com/fireflylearning/pattern-library/pull/170/files#diff-a093c827be0330a4429cba7cdd39654dR15 in this file. Are both of these intended to exist and if so what's the difference between them?